content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -4,245 +4,219 @@ reviewers:
- * **ABAC** - Attribute-based access control (ABAC) defines an access control paradigm whereby access rights are granted to users through the use of policies which combine attributes together. The policies can use any type of attributes (user attributes, resource attributes, object, environment attributes, etc). To learn more about using the ABAC mode, see [ABAC Mode](/docs/reference/access-authn-authz/abac/).
- * **RBAC** - Role-based access control (RBAC) is a method of regulating access to computer or network resources based on the roles of individual users within an enterprise. In this context, access is the ability of an individual user to perform a specific task, such as view, create, or modify a file. To learn more about using the RBAC mode, see [RBAC Mode](/docs/reference/access-authn-authz/rbac/)
-   * When specified RBAC (Role-Based Access Control) uses the `rbac.authorization.k8s.io` API group to drive authorization decisions, allowing admins to dynamically configure permission policies through the Kubernetes API.
-   * To enable RBAC, start the apiserver with `--authorization-mode=RBAC`.
- * **Webhook** - A WebHook is an HTTP callback: an HTTP POST that occurs when something happens; a simple event-notification via HTTP POST. A web application implementing WebHooks will POST a message to a URL when certain things happen. To learn more about using the Webhook mode, see [Webhook Mode](/docs/reference/access-authn-authz/webhook/).
-
-#### Checking API Access
-
-`kubectl` provides the `auth can-i` subcommand for quickly querying the API authorization layer.
-The command uses the `SelfSubjectAccessReview` API to determine if the current user can perform
-a given action, and works regardless of the authorization mode used.
+Kubernetes expects attributes that are common to REST API requests. This means
+that Kubernetes authorization works with existing organization-wide or
+cloud-provider-wide access control systems which may handle other APIs besides
+the Kubernetes API.
 
+## Authorization modes {#authorization-modules}
 
-```bash
-kubectl auth can-i create deployments --namespace dev
-```
+The Kubernetes API server may authorize a request using one of several authorization modes:
 
-The output is similar to this:
+`AlwaysAllow`
+: This mode allows all requests, which brings [security risks](#warning-always-allow). Use this authorization mode only if you do not require authorization for your API requests (for example, for testing).
 
-```
-yes
-```
+`AlwaysDeny`
+: This mode blocks all requests. Use this authorization mode only for testing.
 
-```shell
-kubectl auth can-i create deployments --namespace prod
-```
-
-The output is similar to this:
+`ABAC` ([attribute-based access control](/docs/reference/access-authn-authz/abac/))
```

Comment:
ABAC should not been formatted as code, I think.

Reply:
The other modes are all formatted as teletype, because these are specific strings you set. See for example `Node` on line 132.

Reply:
If I change it, I'd change all of them and not just ABAC.

Reply:
BTW, this might be easier to review by commit. Have a look at the commit that this line changed in; it's a lot easier to read IMO.